server/app/driver/driver.py

```diff
@@ -2,8 +2,27 @@
 
 
 class CXGDriver(metaclass=ABCMeta):
-    def __init__(self, data, schema=None, graph_method=None, diffexp_method=None):
+    def __init__(self, data, graph_method=None, diffexp_method=None):
         self.data = self._load_data(data)
+        self._set_features(graph_method=graph_method, diffexp_method=diffexp_method, cluster=None)
```

Comment:
rather than do this, i would just make a `features` `@property` that calculates this and returns it.

rationale: keep your constructor lightweight, and this postpones the work until it's needed.

Reply:
Cool, can do!